### ISSUE TITLE
iMac w/ case sensitive FS errors when using binaries

### ISSUE THREAD

**Issue by zbeekman** [ActivityWatch/activitywatch#222]
I downloaded and unzipped this binary: https://github.com/ActivityWatch/activitywatch/releases/download/v0.8.0b6/activitywatch-v0.8.0b6-macos-x86_64.zip

(i.e., AW version is v0.8.0b6)

I put aw into my home directory, and tried to run the binaries:

```
$ ./aw-qt
Fatal Python error: initfsencoding: unable to load the file system codec
zipimport.ZipImportError: can't find module 'encodings'

Current thread 0x00007fff8d417380 (most recent call first):
Abort trap: 6
```

No log file was produced at ~/Library/Logs/activitywatch

Any ideas?

**Comment by ErikBjare** [comment]
I was going to ask for your macOS version, and say that I have no clue.

Then I got off my phone, and found this: https://github.com/pyinstaller/pyinstaller/issues/3219

Apparently, the macOS CI builds have sneakily upgraded to Python 3.7 because we didn't have the version locked down, which broke PyInstaller that doesn't support Python 3.7 (and we should be building on Python 3.5).

**Comment by zbeekman** [comment]
Great, glad you pinned down the issue! I'm on macOS 10.13.6 (iMac late 2014) if that helps at all...

So do we need to wait until upstream fixes this, or can we pin the version in requirements.txt or similar and cut a new release or release asset?

**Comment by zbeekman** [comment]
Or does the travis.yml need to specify the python version for macOS builds?

**Comment by ErikBjare** [comment]
The second, it's a minor fix in `.travis.yml`. I'll get you a build with a fix that you can test soon.

**Comment by ErikBjare** [comment]
Building in https://github.com/ActivityWatch/activitywatch/pull/223 now, I'll post a link to the download tomorrow.

**Comment by ErikBjare** [comment]
@zbeekman Try this build, it should work: https://s3.amazonaws.com/activitywatch-builds/activitywatch-7bc3bbe-macos-x86_64.zip

**Comment by zbeekman** [comment]
Seems like it's working. I ran `./aw-qt` and iTerm asked for permission to control my computer. After granting that, I restarted and can browse to localhost:5600! Woot! Thanks so much!

**Comment by ErikBjare** [comment]
@zbeekman Great that it worked! Thanks for reporting and testing.

**Comment by VirenMohindra** [comment]
Is https://s3.amazonaws.com/activitywatch-builds/activitywatch-7bc3bbe-macos-x86_64.zip the latest official release for Mac OS?

`v0.8.0b6` doesn't seem to work while the above link does -- maybe it should get updated so users don't have to go rifling through closed issues?

**Comment by harshhpareek** [comment]
The above links have stopped working, but for anyone else who's here, you can grab one of the files in the <key>...</key> tags at [https://s3.amazonaws.com/activitywatch-builds/](https://s3.amazonaws.com/activitywatch-builds/) 

The files are at https://s3.amazonaws.com/activitywatch-builds/<key>, e.g. I used this one: [https://s3.amazonaws.com/activitywatch-builds/activitywatch-8b4b625-macos-x86_64.zip](https://s3.amazonaws.com/activitywatch-builds/activitywatch-8b4b625-macos-x86_64.zip), but it'll be a dead link soon

**Comment by ErikBjare** [comment]
@harshhpareek Thanks! Sorry for still not having released v0.8.0b7, I'll try and get around to it in a week or two when my exams are over.

**Comment by ErikBjare** [comment]
I released v0.8.0b7 today which should work fine on macOS. Sorry for the delay.
